### DISCUSSION TITLE
SABnzbd Wizard: turn on SSL Checkbox by default?

### DISCUSSION THREAD

**Discussion by sanderjo** [sabnzbd/sabnzbd#2111] [Ideas]
In the SABnzbd Wizard: would it be a good idea to turn the SSL checkbox on by default?

![image](https://user-images.githubusercontent.com/1273502/153917777-a1da6fa4-40ec-4b9e-844d-8e2a82595095.png)

**Comment by Safihre** [comment]
Only after this is fixed in Python: https://github.com/python/cpython/pull/25478

**Comment by sanderjo** [comment]
So close this?
Or keep open to monitor it, and the python fix ... ?
